### PR TITLE
Add script to combine data

### PR DESCRIPTION
**Pull request by victoriademan**
Hi everyone,

This script links to issue #39 and combines the cumulative moisture sources for all models in one netcdf file. This file is much smaller and easier to read and use in your scripts instead of the whole code.

### CODE REVIEW

**Review by chrisweij — APPROVED** [review]
Looks a very good step for me. There is a small consistency in how to read in WAM2layers data for the different cases (see comment), and further checks for other models might need to be done, but I suggest to approve this for now.

**Review by Peter9192 — APPROVED** [review]
I just tried the script on my machine, and that works like a charm. All I had to do was change the path on line 30.

I guess the coordinates for lagranto still need to be swapped though, but after that I think we should merge this :-)

File: combine_data.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+ds_flexpart_xu = xr.open_dataset(basedir+"results Ru_Xu_FLEXPART/e_daily.nc").rename(longitude='lon', latitude='lat')
+srcs_flexpart_xu = ds_flexpart_xu["variable"].sum("time")
+
+########################################################
+## Lagranto CHc                                       ##
+########################################################
+ds_lagranto_CHc = xr.open_mfdataset(basedir+"results CHc LAGRANTO/Pakistan_2022_CHc_*.nc", combine = 'nested', concat_dim='time')
+srcs_lagranto_CHc = ds_lagranto_CHc["N"].sum("time").squeeze().rename(dimy_N='lat', dimx_N='lon')
+srcs_lagranto_CHc.assign_coords(lat=srcs_TRACMASS.lat, lon=srcs_TRACMASS.lon)
+srcs_lagranto_CHc = srcs_lagranto_CHc.assign_coords(lat=srcs_TRACMASS.lat, lon=srcs_TRACMASS.lon)
```
[the remark this answers is not in the document]
Comment (Peter9192) [reply combine_data.py:161]: This I can see :-)
Now I think you can also "commit suggestion", right?

**Comment by Peter9192** [comment]
Great, I'd say go ahead and merge it now. Perhaps we can open a new issue about the WAM2layers unit conversions

**Pull request merged**